src/processor.cpp

```diff
@@ -10,10 +14,6 @@
 #include <map>
 #include <algorithm>
 
-#include "llvm-version.h"
-#include <llvm/ADT/StringRef.h>
-#include <llvm/Support/MathExtras.h>
-
```

Comment:
Is this actually necessary or an accidentally included change?

Reply:
I'm using llvm in processor.h now. In general, anyways we usually do those headers before "julia.h"